go.mod

```diff
@@ -164,7 +164,7 @@ replace github.com/ethereum/go-ethereum => github.com/crypto-org-chain/go-ethere
 // TODO: remove when ibc-go and ethermint upgrades cosmos-sdk
 replace github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.44.7-0.20220214161517-8a26cd10b4be
 
-replace github.com/tharsis/ethermint => github.com/crypto-org-chain/ethermint v0.7.2-cronos-9
+replace github.com/tharsis/ethermint => github.com/crypto-org-chain/ethermint v0.7.2-cronos-10
```

Comment:
https://github.com/crypto-org-chain/ethermint/compare/v0.7.2-cronos-9...v0.7.2-cronos-10?expand=1

Reply:
https://github.com/crypto-org-chain/ethermint/compare/v0.7.2-cronos-9...v0.7.2-cronos-10?expand=1#diff-7f75fec163267a96892043b684f88d495897981a15943b06e4b11e5458c62b8cR347
uses enum or const var better than the hardcode style in here.

Reply:
> https://github.com/crypto-org-chain/ethermint/compare/v0.7.2-cronos-9...v0.7.2-cronos-10?expand=1#diff-7f75fec163267a96892043b684f88d495897981a15943b06e4b11e5458c62b8cR349 Also, do we guarantee the following txs in the txResults (if it has) will be the same `OOG` status?

yes, by the definition block gas limit, it shouldn't execute other txs after exceeded.

Reply:
> https://github.com/crypto-org-chain/ethermint/compare/v0.7.2-cronos-9...v0.7.2-cronos-10?expand=1#diff-7f75fec163267a96892043b684f88d495897981a15943b06e4b11e5458c62b8cR347 (minor) uses enum or const var better than the hardcode style in here.

good point, it should be `sdkerrors.ErrOutOfGas.ABCICode()`, but since the ethermint version is already tagged, maybe we can live with the current one?

Reply:
exit gas accumulation when txResult gets out-of-block-gas error, LGTM